src/utils/src/lib.rs

```diff
@@ -60,5 +60,5 @@ fn rand_bytes_impl(rand_fn: &dyn Fn() -> u32, len: usize) -> Vec<u8> {
 
 /// Get a pseudo random vector of length `len` with bytes.
 pub fn rand_bytes(len: usize) -> Vec<u8> {
```

Comment:
Note: We should have a follow up PR to use the upstream `rand_bytes()` from the new version of vmm-sys-util, instead of this one in `lib.rs` that was defined for temporary use.

Reply:
resolving this so I can merge. Can you open an issue for it instead?